src/lua/datetime.lua

```diff
@@ -42,7 +42,7 @@ typedef enum {
 } dt_dow_t;
```

Comment:
> datetime: simplify boundary checks
We used to use very ugly and tricky to check that passed years. months
and day are not exceeding supported values.

Two typos in the last commit message:

very ugly and tricky to check <- tricky what?
passed years. months <- dot instead of a comma.